inc/mlx90632.h

```diff
@@ -324,6 +339,21 @@ void mlx90632_set_emissivity(double value);
  */
 double mlx90632_get_emissivity(void);
 
+/** Sets the refresh rate of the sensor using the MLX90632_EE_MEAS_1 and MLX90632_EE_MEAS_2 registers
+ *
+ * @param[in] MLX90632_MEAS enum refresh rate to set
+ *
+ * @retval <0 Something went wrong. Consult errno.h for more details.
+ */
+int32_t mlx90632_set_refreshRate(MLX90632_MEAS measRate);
```

Comment:
Please make function names lowercase with underscore as separator (just to keep the file aligned)
```suggestion
int32_t mlx90632_set_refresh_rate(mlx90632_meas_t measRate);
```